index.js

```diff
@@ -290,6 +349,17 @@ const ContainerService = {
     ss.storage.identitiesData = this._identitiesState;
   },
 
+  sendTelemetryPayload(params = {}) {
+    // when pings come from popup, delete "method" prop
+    delete params.method;
+    let payload = { // eslint-disable-line prefer-const
+      "uuid": this._metricsUUID
+    };
+    Object.assign(payload, params);
+
+    this._sendEvent(payload);
```

Comment:
Yeah I'll try to consolidate as much complexity into a single place as I can where the payloads are similar enough.